src/Map/main.vala

```diff
@@ -1,15 +1,16 @@
-#! /usr/bin/env -S vala workbench.vala --pkg libadwaita-1 --pkg shumate-1.0
+#! /usr/bin/env -S vala workbench.vala --pkg libadwaita-1 --pkg gstreamer-1.0 --pkg libportal --pkg shumate-1.0
```

Comment:
Is there any reason why gstreamer and libportal are needed for this demo?

Reply:
My bad, forgot to remove them